integration/init-test/in_docker_test_script.sh

```diff
@@ -11,4 +11,4 @@ echo "Installing reflex from local repo code"
 cd /reflex-repo
 poetry install
 echo "Running reflex init in test project dir"
-poetry run /bin/bash -c "cd ~/hello && reflex init"
\ No newline at end of file
+poetry run /bin/bash -c "cd ~/hello && reflex init && rm -rf ~/.reflex .web && reflex export --backend-only"
```

Comment:
The internal logic depends on platform (like windows vs Linux etc.). Would it make sense to also inject this into the example/counter or reflex-web integration tests?  I.e.

Prior to running the normal `reflex init` in those integration tests, just try to `export --backend-only` (and possibly `reflex run --backend-only`?  That way we gain full matrix coverage for this behavior.